routes/api.php

```diff
@@ -24,3 +24,10 @@
 Route::post('/register', [UserController::class, 'register']);
 Route::put('/users/{userId}', [UserController::class, 'editUser']);
 Route::post('/login', [UserController::class, 'loginUser']);
+Route::post('/logout', [UserController::class, 'logout'])->middleware('auth:sanctum');
+Route::put('/users/{userId}/change-password', [UserController::class, 'changePassword'])->middleware('auth:sanctum');
```

Comment:
here the url shpuld be like this
Route::put('/users/change-password', [UserController::class, 'changePassword'])->middleware('auth:sanctum');